hive/agents/__init__.py

```diff
@@ -7,6 +7,7 @@
 from hive.agents.rainbow import RainbowDQNAgent
 from hive.agents.random import RandomAgent
 from hive.agents.td3 import TD3
+from hive.agents.deep_dyna_q import DeepDynaQ
```

Comment:
imports should be alphabetized. Make sure you run the isort or some type of import sorting tool